### ISSUE TITLE
Add labels to "From" and "To" fields in advenced search for float and date metadata fields

### ISSUE THREAD

**Issue by wfenrich** [CeON/fairchive#828]
Currently we show the description in only in placehoders, which is insufficient from WCAG perspective.
